build.gradle

```diff
@@ -15,3 +17,22 @@ task wrapper(type: Wrapper) {
 dependencies {
   testCompile 'junit:junit:4.12'
 }
+
+publishing {
+  repositories {
+    maven {
+      url "s3://casetext-ds-code-deploys/maven"
+      credentials(AwsCredentials) {
+        accessKey "$System.env.AWS_ACCESS_KEY_ID"
+        secretKey "$System.env.AWS_SECRET_ACCESS_KEY"
+      }
+    }
+  }
+
+  publications {
+    fnvProfileSignature(MavenPublication) {
+      artifactId 'fnv-profile-signature'
```

Comment:
The name of the published artifact has been `fnv-text-profile-signature` until now (e.g., [crick#build.gradle:46](/casetext/crick/blob/master/build.gradle#L46)). Are you intending to change the name here?